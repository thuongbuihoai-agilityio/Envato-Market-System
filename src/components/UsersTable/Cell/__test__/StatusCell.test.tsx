import { render } from '@testing-library/react';
import { StatusCell } from '..';
import { Table, Tr } from '@chakra-ui/react';

describe('TotalCard component', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Table>
        <Tr>
          <StatusCell />
        </Tr>
      </Table>,
    );

    expect(container).toMatchSnapshot();
  });
});
