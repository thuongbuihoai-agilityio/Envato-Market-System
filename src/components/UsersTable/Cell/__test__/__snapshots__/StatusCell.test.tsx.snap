// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TotalCard component renders correctly 1`] = `
<div>
  <table
    class="chakra-table css-0"
  >
    <tr
      class="css-0"
    >
      <td
        class="css-dq40i2"
      >
        <span
          class="chakra-badge css-3jdwwr"
        />
      </td>
    </tr>
  </table>
</div>
`;
